backend/blueprint/api/controllers/generateBadges.py

```diff
@@ -1,11 +1,35 @@
-from flask import Blueprint, jsonify
+import os
+from flask import Blueprint, jsonify, request
+from flask import current_app as app
+# from api.helpers.verifyToken import loginRequired
 from api.utils.response import Response
+from api.utils.svg_to_png import SVG2PNG
+from api.utils.merge_badges import MergeBadges
+
 
 router = Blueprint('generateBadges', __name__)
 
 
 @router.route('/generate_badges', methods=['POST'])
 def generateBadges():
+    try:
+        data = request.get_json()
+    except Exception as e:
+        return jsonify(
+            Response(401).exceptWithMessage(
+                str(e),
+                'Could not find any JSON'))
+
+    csv_name = data['csv']
```

Comment:
please check for the availability of minimum required parameters and raise required error responses